app/views/adoption_requests/new.html.erb

```diff
@@ -1,4 +1,5 @@
-<% provide(:title, "Adoption Request Form") %>
+<% page_title! 'Adoption Request Form' %>
```

Comment:
Thats the gem RubyGems.org use to manage their page title. I found it easier to add the gem that to replace it.